src/ObjectStorage.php

```diff
@@ -55,11 +55,7 @@ public function get($object)
     {
         $hash = spl_object_hash($object);
 
-        if (isset($this->data[$hash])) {
-            return $this->data[$hash];
-        }
-
-        return null;
+        return $this->data[$hash] ?? null;
```

Comment:
You can keep this change alone.

Reply:
How about reverting this due to your previous comment.

Reply:
This is an `isset($x) ? $x : null`, so conceptually equivalent to `$x ?? null`. I can accept this change.